data/wanderers start.txt

```diff
@@ -376,16 +376,59 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <system>, where repairs are needed from Unfettered attacks."
+	passengers 1
+	cargo "building supplies" 20
+	source 
+		near "Sich'ka'ara" 1 100
+	destination "Vara Ke'sok"
+	to offer
+		random < 60
+		has "Wanderers Conversation: offered"
+		not "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		conversation
+			`Walking around the spaceport, you are approached by two Wanderers. One of them begins speaking in the Hai language, which your translator picks up. "Greetings, [stranger, curiosity]. We heard you perform [charitable, heroic] acts. Is this correct?"`
+			choice
+				`	"Yes it is. What do you need?"`
+				`	"That sounds like me."`
```

Comment:
I don't get the sense that we need these two choices that lead to the same outcome since we already have a third choice that leads to a different outcome.

Reply:
Removed the unnecessary choice.